test/parallel/test-module-isBuiltIn.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { isBuiltIn } = require('module');
+
+// Includes modules in lib/ (even deprecated ones)
+assert(isBuiltIn('http'));
+assert(isBuiltIn('sys'));
+assert(isBuiltIn('node:fs'));
+assert(isBuiltIn('node:test'));
+
+// Does not include internal modules
+assert(!isBuiltIn('internal'));
```

Comment:
```suggestion
assert(!isBuiltIn('internal/errors'));
```

`'internal'` alone is not even an internal module